tests/test_integration.py

```diff
@@ -124,7 +124,7 @@ def test_build(monkeypatch, project, args, call, tmp_path):
 
 def test_isolation(tmp_dir, package_test_flit, mocker):
     try:
-        import flit_core  # noqa: F401  # imported but unused
```

Comment:
Is this expected? It's technically correct - `flit_core` is not used. I don't see a changelog entry or PR mentioning a change to F401.

The correct fix for this always has been to use `importlib.util.find_spec`, which is also faster.